### PR TITLE
Add silentcodeg to MAINTAINERS

### PR DESCRIPTION
**Pull request by silentcodeg**
[list of PRs](https://github.com/go-gitea/gitea/pulls?q=is%3Apr+author%3Asilentcodeg+is%3Aclosed)

### CODE REVIEW

**Comment by 6543** [comment]
@silentcodeg are you on discord?

**Comment by silentcodeg** [comment]
I am on matrix.

**Review by 6543 — APPROVED** [review]
looking forward to you contirbs

**Comment by 6543** [comment]
I think you are already aware of https://matrix.to/#/#gitea-space:matrix.org

the maintainer chat is currently on discord only ... I might can create some bridging but expect it in the future

**Review by delvh — APPROVED** [review]
Isn't there a `g` missing?

File: MAINTAINERS [hunk 1]
```diff
@@ -46,3 +46,4 @@ Jimmy Praet <jimmy.praet@telenet.be> (@jpraet)
 Leon Hofmeister <dev.lh@web.de> (@delvh) 
 Gusted <williamzijl7@hotmail.com) (@Gusted)
 singuliere <singuliere@autistici.org> (@singuliere)
+silentcode <silentcode@senga.org> (@silentcode)
```
Comment (6543) [inline MAINTAINERS]:
```suggestion
silentcodeg <silentcode@senga.org> (@silentcodeg)
```

Comment (6543) [reply MAINTAINERS]: ?

**Review by silverwind — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Comment by 6543** [comment]
:wave: @silentcodeg welcome in the team :) can you point me to your matrix id?

**Comment by silentcodeg** [comment]
My matrix id is @silentcodeg thank you.

**Comment by silentcodeg** [comment]
Can you add me to the GitHub maintainers team?

**Comment by lunny** [comment]
@silentcodeg invited!!!

**Pull request merged**